src/bp/core/dialog/flow/flow-service.ts

```diff
@@ -88,7 +88,7 @@ export class FlowService {
   private _listenForCacheInvalidation() {
     this.cache.events.on('invalidation', async key => {
       try {
-        const matches = key.match(/^([A-Z0-9-_]+)::data\/bots\/([A-Z0-9-_]+)\/flows\/([\s\S]+(flow|ui)\.json)/i)
+        const matches = key.match(/^([A-Z0-9-_]+)::data\/bots\/([A-Z0-9-_]+)\/flows\/([\s\S]+(flow)\.json)/i)
```

Comment:
The only issue I see with this change is that you cannot edit a `*.ui.json` file directly (when running BPFS_STORAGE=disk) but you can for a `*.flow.json` file.

Reply:
Have you ever edited manually a .ui.json file? I haven't. The performance increase we get is much more valuable than the lost functionality, and we can even propose a workaround if someone need to change it manually: simply re-save the .flow.json file.

When you have 3000 .flow.json and 3000 .ui.json, there really is a noticeable effect when you have to parse 6000 files twice :-P

Reply:
> Have you ever edited manually a .ui.json file? I haven't. The performance increase we get is much more valuable than the lost functionality, and we can even propose a workaround if someone need to change it manually: simply re-save the .flow.json file.
> 
> When you have 3000 .flow.json and 3000 .ui.json, there really is a noticeable effect when you have to parse 6000 files twice :-P

I totally agree with you I just wanted to make sure we didn't regress for some users with this change!